recipes/eventio/meta.yaml

```diff
@@ -0,0 +1,58 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Please remove these comments. Also, take a look at grayskull, it will generate a recipe that is "almost" ready for merge automatically.